service_mesh/v1x/ossm-custom-resources.adoc

```diff
@@ -8,7 +8,7 @@ You can customize your {ProductName} by modifying the default {ProductShortName}
 
 == Prerequisites
 * An account with the `cluster-admin` role.
-* Completed the xref:../../service_mesh/v1x/preparing-ossm-installation.adoc#preparing-ossm-installation-v1x[Preparing to install {ProductName}] process.
+* Completed the xref:../../service_mesh/v1x/preparing-ossm-installation.adoc#preparing-ossm-installation-v1x[Preparing to install {ProductName}] procevss.
```

Comment:
s/procevss/process

Reply:
How on earth?  Must have been typing in the wrong window.  Fixed!